Assets/XML/Units/CIV4UnitInfos.xml

```diff
@@ -11106,12 +11106,6 @@
 					<PassableTech>TECH_OPTICS</PassableTech>
 				</TerrainPassableTech>
 			</TerrainPassableTechs>
-			<FeaturePassableTechs>

```

Comment:
True, but then you also need to remove the `FeatureImpassables` entry for capes.